pmda/rmsf.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
```

Comment:
Add your code to file [pmda/rms.py](https://github.com/MDAnalysis/pmda/blob/master/pmda/rms.py) to mirror `MDAnalysis.analysis.rms.RMSF`.

Reply:
This will also set the correct `from __future__ import ...` and you can just add your docs to it (following the example of the existing ones.)